htdocs/Frameworks/moduleclasses/moduleadmin/moduleadmin.php

```
@@ -588,14 +588,44 @@ public function renderAbout($business = '', $logo_xoops = true)
+        $ret .= $this->renderFooterInfo($logo_xoops);
+        return $ret;
+    }
+
+    /**
+     * Create HTML text to display the standard admin footer
+     *
+     * @param bool $logo_xoops true to display XOOPS logo and link
+     *
+     * @return string HTML to display
+     */
+    public function renderFooterInfo($logo_xoops = true)
```

PHPMD (2.15.0)

[error] The method renderFooterInfo has a boolean flag argument $logo_xoops, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)